### PR TITLE
Feature/two stage type creation

### PR DESCRIPTION
**Pull request by NathHorrigan**

### CODE REVIEW

File: grapple/actions.py [hunk 1]
```diff
@@ -128,47 +134,81 @@ def build_node_type(
...
-    # types.
-    type_meta["Meta"].exclude_fields = exclude_fields
-    graphql_node = type(type_name, (base_type,), type_meta)
-
-    for name, method in methods.items():
-        setattr(graphql_node, name, MethodType(method, graphql_node))
-
-    return graphql_node
+    # Create a tempory model and tempory node that will be replaced later on.
+    class StudModel(models.Model):
```
Comment (zerolab) [inline grapple/actions.py]: nitpick: `Stud` → `Stub` 😂

Comment (NathHorrigan) [reply grapple/actions.py]: Amazing! Thank You Dan.

Comment (NathHorrigan) [reply grapple/actions.py]: `hotfix/fix-foggy-head` :joy:

Comment (zerolab) [reply grapple/actions.py]: 🤣

**Pull request merged**